sdks/python/apache_beam/runners/interactive/extensions/apache-beam-jupyterlab-sidepanel/style/mdc-theme.css

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/*
+ * Theme configured from https://rmwc.io/.
+ */
+:root {
+  --mdc-theme-primary: #1a73e8;
```

Comment:
We ran into this issue with some of our extensions this summer and found that it was best if we referenced JupyterLab's CSS variables for colors where possible so we would be able to support users with different JupyterLab themes.

- https://jupyterlab.readthedocs.io/en/stable/developer/css.html
- https://github.com/jupyterlab/jupyterlab/blob/master/packages/theme-light-extension/style/variables.css

Reply:
Thanks! This is really helpful. I've changed to use the CSS variables defined by Jupyter.